packages/lit-html/src/lit-html.ts

```diff
@@ -422,10 +614,19 @@ export const render = (
   container: HTMLElement | DocumentFragment,
   options?: RenderOptions
 ): RootPart => {
+  const renderId = DEV_MODE ? debugLogId++ : 0;
   const partOwnerNode = options?.renderBefore ?? container;
   // This property needs to remain unminified.
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   let part: ChildPart = (partOwnerNode as any)['_$litPart$'];
+  debugLogEvent?.({
```

Comment:
Are Terser and Closure smart enough to remove these statements entirely when debugLogEvent is undefined?

Reply:
Yep!